### PR TITLE
MP load purchase history

### PR DESCRIPTION
**Pull request by pattmax00**

### CODE REVIEW

File: AcademicReward/Database/PurchaseHistoryProfileRelationship.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿using AcademicReward.ModelClass;
+using Npgsql;
+using System.Collections.ObjectModel;
+
+namespace AcademicReward.Database
+{
+    internal class PurchaseHistoryProfileRelationship : AcademicRewardsDatabase
```
Comment (Wil-LaLonde) [inline AcademicReward/Database/PurchaseHistoryProfileRelationship.cs:7]: Can you add the comment block to the top of this new class?

File: AcademicReward/Views/PurchaseHistoryPage.xaml.cs [hunk 2]
```diff
@@ -1,19 +1,30 @@
+using AcademicReward.Database;
+using AcademicReward.ModelClass;
 using System.Collections.ObjectModel;
 
 namespace AcademicReward.Views;
 
 /// <summary>
-/// PurchaseHistoryPage shows all purchased items for a profile
```
Comment (Wil-LaLonde) [inline AcademicReward/Views/PurchaseHistoryPage.xaml.cs:6]: Could we keep these comments? I'm sure you didn't mean to get rid of them, unless...did you?

Comment (pattmax00) [reply AcademicReward/Views/PurchaseHistoryPage.xaml.cs:6]: I think this was a merge conflict I must've removed by mistake, I'll add it back

**Comment by pattmax00** [comment]
@Wil-LaLonde Should be good now

**Review by Wil-LaLonde — APPROVED** [review]
Looks good

**Pull request merged**